modules/ztp-image-based-upgrade-prep.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-prep_{context}"]
+= Preparing the {sno} clusters for the image-based upgrade
```

Comment:
General Q: is this feature only for SNO clusters? Can it be used with for eg., 3 node clusters, standard clusters, etc.

Reply:
Yes, it's limited to SNO clusters only